pkg/util/tracing/tracingpb/recorded_span.proto

```diff
@@ -76,6 +76,9 @@ message RecordedSpan {
   //
   // TODO(tbg): rename once DeprecatedStats is removed.
   repeated google.protobuf.Any internal_structured = 11;
+
+  // Goroutine ID associated with this span at time of span creation.
```

Comment:
[nit] Here, and above, I'd phrase it as "ID of the goroutine creating the span."